lib/server.js

```diff
@@ -196,7 +205,7 @@ module.exports = function createServer(config, log) {
       fetchRecords(ip, email, phoneNumber)
         .spread(
           function (ipRecord, reputation, emailRecord, ipEmailRecord, smsRecord) {
-            if (ipRecord.isBlocked()) {
+            if (ipRecord.isBlocked() && ! isWhitelisted(ip, email)) {
```

Comment:
I personally always find the dangling bang odd, as if it's unloved. Grumble mumble spaces vs tabs code styles ignore me.